ghost/webmentions/lib/MentionSendingService.js

```diff
@@ -0,0 +1,163 @@
+const errors = require('@tryghost/errors');
+const logging = require('@tryghost/logging');
+
+module.exports = class MentionSendingService {
+    #discoveryService;
```

Comment:
out of interest, I've not seen the `#` pattern before, what is it?

Reply:
# marks a private attribute or method, refs https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Classes/Private_class_fields

We used to also have `_` before because Node only supports this from 12.0. Sometimes we also just mark something as `@private` with JSDocs. Sometimes we cannot really use `#` because those methods are also not accessible for unit testing (haven't found a way around that yet)